.github/workflows/linting_bash_python_yaml_files.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
     - name: Set up changed files
       id: changed_files
       run: |
-        git diff --name-status origin/master...HEAD | grep -E '^[AM].*\.(yaml|yml)$' | awk '{print $2}' > changed_files_in_PR.txt || true
+        git diff --name-status origin/master...HEAD | grep -E '^[AM].*\/(common|example)\/.*\.(yaml|yml)$' | awk '{print $2}' > changed_files_in_PR.txt || true
```

Comment:
can you explain a bit? doe sthis only check /common/* and /example/* for now?

Reply:
Yes. as I remember Before we discussed doing that. That's why I added It. Any concerns regarding that ?

Reply:
Yes, please add .github and tests and hack next to common and example

Reply:
but please fix this in a follow up PR.

Reply:
Add those files as well